fedimint-server/src/net/peers.rs

```diff
@@ -81,12 +81,46 @@ struct PeerConnectionStateMachine<M> {
     state: PeerConnectionState<M>,
 }
 
+#[derive(Debug, Clone, Copy)]
+pub struct DelayCalculator {
+    scaling_factor: f64,
+}
+
```

Comment:
nit: Could use some docs.

Really nice abstraction, eventually we might even want to expose it in the config for people to choose reconnect strategies and timeouts.

Reply:
Added minimal documentation